erpnext/stock/doctype/item/item.js

```diff
@@ -119,6 +119,50 @@ frappe.ui.form.on("Item", {
 		erpnext.item.weight_to_validate(frm);
 	},
 
+	image_preview: function(frm){
+
+		if (frm.doc.list_all_image.length == 0)
+		{
+			frappe.msgprint(__("Images not avalilable !!"));
+		}
+		else if (frm.doc.list_all_image){
+				var image_path = frm.doc.image;
+				fields = []
+				for(var i=0;i< frm.doc.list_all_image.length;i++){
+//					fieldname = 'image'
+//					fieldname = 'title'
+//					fieldname = 'description'
+				    image_dict = {
```

Comment:
why this modal?

Also don't use variable names like `image_dict1` etc. This should just be one list of dicts?